static/js/reducers/index.js

```diff
@@ -54,9 +55,10 @@ export const profiles = (state: ProfileState = INITIAL_PROFILES_STATE, action: A
       getStatus: FETCH_PROCESSING
     });
   case RECEIVE_GET_USER_PROFILE_SUCCESS:
+    profile = cloneAndReplace(action.payload.profile, v => v === null ? "" : v);
```

Comment:
I feel like this may have unintended consequences because each null does not necessarily replace a string type, though there are no profile fields like that right now. We are telling the type checker that these fields are not null, but we're just replacing the null with an empty string sentinel value.